spec/index.bs

```diff
@@ -86,12 +86,15 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
 
   <em>This section is non-normative.</em>
 
-  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away). This makes it difficult to synchronize client data from web apps (such as photo uploads, document changes, or composed emails) with servers. If the browser closes or the user navigates away before synchronization can complete, the app must wait until the user revisits the page to try again. This specification provides a new onsync <a>service worker</a> event which can fire <a>in the background</a> so that synchronization attempts can continue despite adverse conditions when initially requested. This API is intended to reduce the time between content creation and content synchronization with the server.
+  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away). This makes it difficult to synchronize client data from web apps (such as photo uploads, document changes, or composed emails) with servers. It also makes it difficult to synchronize updated content on the servers with the web ap. If the browser closes or the user navigates away before synchronization can complete, the app must wait until the user revisits the page to try again. This specification provides new onsync and onperiodicsync <a>service worker</a> events which can fire <a>in the background</a> so that synchronization attempts can continue despite adverse conditions when initially requested. This API is intended to reduce the time between content creation and content synchronization between the server and the web app.
+
+  Two types of synchronizations are introduced: one-shot, which provides a one-off synchronization opportunity, and periodic,
+  which periodically provides synchronization opportunities.
```

Comment:
move to the line above

Reply:
Does it matter in a .bs file where the line break happens?